webapp/components/Editor/Editor.vue

```diff
@@ -214,8 +215,9 @@ export default {
     },
     sanitizeQuery(query) {
       if (this.suggestionType === HASHTAG) {
-        // remove all not allowed chars
-        query = query.replace(/[^a-zA-Z0-9]/gm, '')
+        // remove all non unicode letters and non digits
+        const regexMatchAllNonUnicodeLettersOrDigits = build('[^\\pL0-9]')
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-09-03T14:05:50Z" title="Tuesday, September 3rd 2019, 4:05:50 pm +02:00">Sep 3, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -212,8 +213,9 @@ export default {
     },
     sanitizeQuery(query) {
       if (this.suggestionType === HASHTAG) {
-        // remove all not allowed chars
-        query = query.replace(/[^a-zA-Z0-9]/gm, '')
+        // remove all unallowed chars
+        const regX = build('[^\\pL0-9]')
```


same as above – I could go check what this regex does exactly but would prefer to see it directly in the code, either as variable name or, since there's a comment already, as explanation in the comment

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-09-04T12:16:08Z" title="Wednesday, September 4th 2019, 2:16:08 pm +02:00">Sep 4, 2019</time>_

---


Wonderful 😍

@mattwr18 did already … 😍
… and I changed the name again for clarification.